test/core/miner/precommit_batcher_test.cpp

```diff
@@ -202,11 +223,14 @@ namespace fc::mining {
     precInf.sealed_cid = "010001020013"_cid;
     si.sector_number = 4;
 
-    EXPECT_OUTCOME_TRUE_1(batcher_->addPreCommit(
-        si, deposit, precInf, [](const outcome::result<CID> &cid) {
-          ASSERT_TRUE(cid.has_value());
-        }));
-    mutual_deposit_ += 10;
+    EXPECT_OUTCOME_TRUE_1(
+        batcher_->addPreCommit(si, deposit, precInf, callback_mock_));
+    ASSERT_TRUE(is_called_);
+
+    is_called_ = false;
+
+    EXPECT_OUTCOME_TRUE_1(
+        batcher_->addPreCommit(si, deposit, precInf, callback_mock_));
     ASSERT_TRUE(is_called_);
   }
-}  // namespace fc::mining
+}  // namespace fc::mining
```

Comment:
Empty line at the end was removed. Please, revert.

Reply:
fixed